modules/bootstrap-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+// * networking/installing-albo-sts-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="nw-bootstra-albo-on-sts-cluster_{context}"]
+= Bootstrapping AWS Load Balancer Operator on Secure Token Service cluster
+
+.Prerequisites
+
+* You must extract and prepare the `coctl` binary.
+
+.Procedure
+
+. Create a directory to store the `CredentialsRequest` of the AWS Load Balancer Operator by running the following command:
++
+[source,terminal]
+----
+$ curl --create-dirs -o <path-to-credrequests-dir>/cr.yaml https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/hack/operator-credentials-request.yaml
+----
+
+. Use the `ccoctl` tool to process `CredentialsRequest` objects of the AWS Load Balancer Operator needed to bootstrap the Operator, by running the following command:
```

Comment:
```suggestion
. Use the `ccoctl` tool to process `CredentialsRequest` objects of the AWS Load Balancer Operator by running the following command:
```
We are already in the `Bootstrapping` chapter, no need to repeat this.

Reply:
acked